### PR TITLE
Added Sundeep Chand alongwith _blank so that sites of participants opens in new tab

### PR DESCRIPTION
**Pull request by SundeepChand**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,29 +38,29 @@ <h2>Participants</h2>
       Check out the amazing sites by participants below!
 
       <div class="sites participants" id="sites">
-        <a href="participants/kat-goff/">Kat Goff</a>
-        <a href="participants/verity-brean/">Verity Brean</a>
```
Comment (linda-lai) [inline index.html:42]: It looks like Verity was removed as a contributor but not added back in, please re-add 🙂

Comment (linda-lai) [reply index.html:42]: Along with `_blank` so that the link will open in a new tab: `<a href="participants/verity-brean/">Verity Brean</a>`

**Comment by linda-lai** [comment]
Aside from change above, all looks good - well done!

File: index.html (hunk 1 above)
Comment (SundeepChand) [reply index.html:42]: > It looks like Verity was removed as a contributor but not added back in, please re-add 🙂

I have fixed it.

Comment (linda-lai) [reply index.html:42]: Excellent, thank you!

**Pull request merged**